### PR TITLE
WIP add `get_results` method for io_helper.py

### PR DESCRIPTION
**Pull request by Marigold**

### CODE REVIEW

File: python-mip/mip_helper/mip_helper/io_helper.py [hunk 1]
```diff
@@ -53,24 +58,50 @@ def fetch_data():
...
-    Store algorithm results in the output DB.
+    Store algorithm results in the output DB. Update results if it already exists.
     :param pfa: PFA formatted results
     :param error: Error message (if any)
     :param shape: Result shape. For example: pfa_json.
     """
     engine = sqlalchemy.create_engine(_get_output_db_url())
 
-    sql = sqlalchemy.text("INSERT INTO job_result VALUES(:job_id, :node, :date, :pfa, :error, :shape, :function)")
+    # TODO: only temporary solution, final one will not be updating existing results
```
Comment (ludovicc) [inline python-mip/mip_helper/mip_helper/io_helper.py]: yes please remove...

File: python-mip/mip_helper/mip_helper/io_helper.py [hunk 2]
```diff
@@ -53,24 +58,50 @@ def fetch_data():
...
+          shape = :shape,
+          function = :function;
+    """
+    sql = sqlalchemy.text(query)
     engine.execute(sql,
                    job_id=_get_job_id(),
                    node=_get_node(),
-                   date=datetime.datetime.now(),
-                   pfa=pfa,
+                   # TODO: shouldn't this rather be utcnow()?
```
Comment (ludovicc) [inline python-mip/mip_helper/mip_helper/io_helper.py]: probably, please try to use utcnow()

Comment (ludovicc) [reply python-mip/mip_helper/mip_helper/io_helper.py]: I have added stats in Marigold-get-results-method branch

Comment (Marigold) [reply python-mip/mip_helper/mip_helper/io_helper.py]: Should I continue with that branch or rebase on top of it? (Can you push directly to my pull request or do I have to give you permissions for that?)

File: python-mip/tests/unit/fixtures.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    # see https://github.com/HBPMedical/woken/blob/master/src/test/resources/metadata/mip_cde_variables.json for
+    # all definitions
+    return {
+        'lefthippocampus': {
+            'code': 'lefthippocampus',
+            'type': 'real',
+            'mean': 3.0,
+            'std': 0.35
+        },
+        'minimentalstate': {
```
Comment (Marigold) [inline python-mip/tests/unit/fixtures.py]: @ludovicc your branch assumes the following format of META definition
```
{
'name': 'lefthippocampus',
'type': {'name': 'real'},
'mean': 3.0,
'std': 0.35
}
```
shouldn't it be rather
```
'name': 'lefthippocampus',
'type': {
    'name': 'real',
    'mean': 3.0,
    'std': 0.35
}
```
to be consistent with nominal types that return
```
{
        'name': 'agegroup',
        'type': {
            'name': 'polynominal',
            'enumeration': ['-50y', '50-59y', '60-69y', '70-79y', '+80y']
        },
        'series': ['70-79y', '70-79y', '70-79y', '70-79y', '+80y']
    }
```
? Thanks!

**Comment by Marigold** [comment]
Closing in favor of https://github.com/LREN-CHUV/python-base-docker-images/pull/2

**Pull request closed without merging**